src/main.c

```diff
@@ -373,28 +373,28 @@ void WriteInterfaces()
         switch (Interface->Type) {
         case PCAPNG_LINKTYPE_ETHERNET:
             if (Interface->IsVMNic) {
-                printf("IF: medium=%s\tID=%u\tIfIndex=%u\tVlanID=%i",
+                printf("IF: medium=vNIC-%s\tID=%u\tIfIndex=%u\tVlanID=%i",
                     Interface->VMNic.SourceNicType,
                     Interface->PcapNgIfIndex,
-                    Interface->VMNic.SourcePortId,
+                    Interface->LowerIfIndex,
                     Interface->VlanId
                 );
                 StringCchPrintfA(
```

Comment:
Maybe I missed this in the last review; why is there a printf followed immediately by a StringCchPrintfA here? Why isn't this just one printf call?

Reply:
It was already present, the first printf is displaying interface information to the console output (see above). The StringCchPrintfA is formatting the interface name, in a more compacted manner, which will be visible in Wireshark (call PcapNgWriteInterfaceDesc).

Reply:
Oh! I'm sorry, I saw "printf" and thought that meant the function prints to the console, but it actually writes to the IfName string.